.kitchen.yml

```diff
@@ -4,79 +4,72 @@ driver:
   provider: parallels
 
 provisioner:
-  product_name: chef
-  product_version: 13.8
+  name: chef_zero
   always_update_cookbooks: true
 
 verifier:
   name: inspec
   sudo: true
+  inspec_tests:
+  - test/integration/default
 
 platforms:
-  - name: apex/macos-10.13.3
-  - name: apex/macos-10.12.6
-  - name: apex/macos-10.11.6
+- name: apex/macos-10.11.6
+- name: apex/macos-10.12.6
+- name: apex/macos-10.13.3
 
 suites:
-  - name: default
-    provisioner:
-      multiple_converge: 2
-      enforce_idempotency: true
-    run_list:
-      - recipe[macos_test::new_users]
-      - recipe[macos::disable_software_updates]
-      - recipe[macos_test::preferences]
-    verifier:
-      inspec_tests:
-        - test/smoke/default
+- name: default
+  run_list:
+  - recipe[macos::disable_software_updates]
+  - recipe[macos_test::preferences]
+  verifier:
+    controls:
+    - dock-appearance
+    - show-all-files
+    - updates-disabled
 
-  - name: keep_awake
-    provisioner:
-      multiple_converge: 2
-      enforce_idempotency: false
-    run_list:
-      - recipe[macos::keep_awake]
-    verifier:
-      inspec_tests:
-        - test/smoke/keep_awake
+- name: power_management
+  run_list:
+  - recipe[macos::keep_awake]
+  verifier:
+    controls:
+    - screensaver-disabled
+    - remote-administration
+    - no-sleep
 
-  - name: machine_name
-    provisioner:
-      multiple_converge: 2
-      enforce_idempotency: true
-    run_list:
-      - recipe[macos_test::machine_name]
-    verifier:
-      inspec_tests:
-        - test/smoke/machine_name
+- name: machine_name
+  run_list:
+  - recipe[macos_test::machine_name]
+  verifier:
+    controls:
+    - nonstandard-computer-name
 
-  - name: xcode
-    run_list:
-      - recipe[macos_test::xcode]
-    verifier:
-      inspec_tests:
-        - test/smoke/xcode
+- name: spotlight
+  run_list:
+  - recipe[macos_test::spotlight]
+  verifier:
+    controls:
+    - indexing/searching
 
-  - name: spotlight
-    provisioner:
-      multiple_converge: 2
-      enforce_idempotency: true
-    run_list:
-      - recipe[macos_test::spotlight]
-    verifier:
-      inspec_tests:
-        - test/smoke/spotlight
+- name: xcode
+  run_list:
+  - recipe[macos_test::xcode]
+  verifier:
+    controls:
+    - xcode/simulators
```

Comment:
These slashes make me thing there's some sort of namespacing happening at first glance - may just `xcode_and_simulators`?